devops/iac/compute/main.tf

```diff
@@ -1,8 +1,18 @@
 module "azurerm_kubernetes_cluster" {
-    source = "../TerraformAKS"
+    source = "../modules/TerraformAKS"
     
 }
 module "azure_acr" {
-    source = "../terraformACR"
+    source = "../modules/terraformACR"
+  
+}
+
+module "service_bus" {
+    source = "../modules/service bus"
+    
+}
```

Comment:
while calling the module from here include the variables here only, for ex add the svcbus name, sku from here, even if default.tf contains the default values